File: official/docs/ruby/address/create_and_verify_address.rb
require 'easypost'

EasyPost.api_key = ENV['EASYPOST_API_KEY']

address = EasyPost::Address.create_and_verify(
  street1: '417 MONTGOMERY ST',
  street2: 'FLOOR 5',
  city: 'SAN FRANCISCO',
  state: 'CA',
  zip: '94104',
  country: 'US',
  company: 'EasyPost',
  phone: '415-123-4567',
)

puts verified_address

File: official/docs/go/address/create_and_verify_address.go
package example

import (
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
    client := easypost.New(apiKey)

	address, err := client.CreateAndVerifyAddress(
        &easypost.Address{
            Street1: "417 Montgomery Street",
            Street2: "FL 5",
            City:    "San Francisco",
            State:   "CA",
            Zip:     "94104",
            Country: "US",
            Company: "EasyPost",
            Phone:   "415-123-4567",
        },
        &easypost.CreateAddressOptions{},
	)

	fmt.Println(address) 
}
